fili-core/src/test/java/com/yahoo/bard/webservice/util/JsonSlurper.java

```diff
@@ -5,6 +5,7 @@
 /*
  * Changed HashMap to LinkedHashMap for consistency. Added ability to optionally sort the lists and maps.
  */
+
```

Comment:
This code was borrowed from Groovy right? While we're here, we really should update the copyright notification to reflect that.